resources.whatwg.org/standard.css

```diff
@@ -304,6 +313,9 @@ dl.domintro dd p:last-child {
 .example::before {
   background: #222222;
 }
+td > .example, td > .note:not(span), td > .warning:not(span) {
+  margin-left: 0.8em;
+}
 td > .example:only-child {
   margin: 0 0 0 0.1em;
   padding: 0;
```

Comment:
This fixes [13.2.2 Parse errors](https://html.spec.whatwg.org/multipage/parsing.html#parse-error-invalid-first-character-of-tag-name), where `.example::before` and `.note::before` intrude into adjacent cells.